frontend/src/components/ScheduleCreation.vue

```diff
@@ -660,12 +651,12 @@ watch(
               @change="toggleZoomLinkCreation"
               class="size-5 rounded-md"
             />
```

Comment:
This can be a separate PR if you want, but we should normalize these checkboxes with the SyncCard's checkboxes. (And then pull those checkboxes into SyncCard 😅 )

Reply:
Ah that is a great suggestion! I forgot we already had styled checkboxes! I did it right away.